rocq failure

Requesting help to fix this failure:

```
File "./rv64d.v", line 9065, characters 35-69:
Error: The variable sys_enable_experimental_extensions was not found
in the current environment.

gmake[3]: *** [model/CMakeFiles/build_rocq_rv64d.dir/build.make:75: rocq/rv64d.vo] Error 1
gmake[2]: *** [CMakeFiles/Makefile2:1002: model/CMakeFiles/build_rocq_rv64d.dir/all] Error 2
gmake[1]: *** [CMakeFiles/Makefile2:1009: model/CMakeFiles/build_rocq_rv64d.dir/rule] Error 2
gmake: *** [Makefile:453: build_rocq_rv64d] Error 2
```

--- a/handwritten_support/riscv_extras.v
+++ b/handwritten_support/riscv_extras.v
@@ -25,6 +25,8 @@ Definition shift_bits_right_arith {a b} (v : mword a) (n : mword b) : mword a :=
 Definition get_time_ns (_:unit) : Z := 0.
 (*declare ocaml target_rep function get_time_ns := `(fun () -> Big_int.of_int (int_of_float (1e9 *. Unix.gettimeofday ())))`*)
 
+Definition sys_enable_experimental_extensions (_:unit) : bool := false.
+
 (* Override the more general version *)
 
 Definition mults_vec {n} (l : mword n) (r : mword n) : mword (2 * n) := mults_vec l r.

